src/components/screens/InfoScreen.jsx

```diff
@@ -1,108 +1,47 @@
-import React, { useState } from "react";
-import {
-  Image,
-  ScrollView,
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View,
-  Modal,
-} from "react-native";
-
-export const InfoScreen = () => {
-  const [isModalVisible, setIsModalVisible] = useState(false);
+import React from "react";
+import { FlatList, StyleSheet, Text, View } from "react-native";
 
+const InfoScreen = () => {
+  const { container, infoTableHeader, infoTableItem } = styles;
   return (
-    <View style={styles.container}>
-      <ScrollView
-        style={styles.container}
-        contentContainerStyle={styles.contentContainer}
-      />
-      <Modal
-        animationType="slide"
-        transparent={false}
-        visible={isModalVisible}
-        onRequestClose={() => {
-          setIsModalVisible(!isModalVisible);
-        }}
-      >
-        <View style={styles.container}>
-          <ScrollView
-            style={styles.container}
-            contentContainerStyle={styles.contentContainer}
-          />
-          <View>
-            <Text style={styles.textInfoMain}>App version</Text>
-            <Text style={styles.textInfo}>1.0</Text>
-
-            <Text style={styles.textInfoMain}>Created by</Text>
-            <Text style={styles.textInfo}>
-              Group D:
-              {"\n"}
-              {"\n"} Patryk Szwed
-              {"\n"} Jakub Zbąski
-              {"\n"} Jarosław Danielak
-              {"\n"} Patryk Kawa
-            </Text>
-
-            <Text style={styles.textInfoMain}>Data provider</Text>
-            <Text style={styles.textInfo}>Safecast</Text>
+    <View>
+      <Text style={infoTableHeader}>APPLICATION DETAILS</Text>
+      <FlatList
+        data={[
+          { key: "App version", value: "1.0" },
+          { key: "Created by", value: "Groupd D" },
+          { key: "Data provider", value: "Safecast" },
+        ]}
```

Comment:
How about moving this array to the top of the file, let's say 3rd line? It will make `return` more clear and easily visible all informations.
Also, how about changing `key` key to `label`?

Reply:
You are right - I moved this array at the beginning of the InfoScreen.jsx file. 
Referring to chaning `key` to `label`, I left it as it was as after changing this field to `label`, warnings about missing key property were being displayed.